nuxt.config.js

```diff
@@ -167,6 +167,7 @@ export default {
     pprURL: process.env.PPR_URL,
     regAccountCreateURL: process.env.REGISTRY_ACCOUNT_CREATE_URL,
     rptURL: process.env.RPT_URL,
```

Comment:
Is this still used/required?

Reply:
Also:
- line 13 in vaults.env
- line 35 in cloudbuild-pr.yaml
- line 13 in .env.example

Reply:
We don't use this URL and same as other products login url. Should we remove them all or keep it?

Reply:
If we don't use it, let's clean it up.

Reply:
I am wrong. We are using it in https://github.com/bcgov/bcregistry/blob/main/utils/user-product-helper.ts